.gitignore

```diff
@@ -18,14 +18,10 @@ jni/CMakeCache.txt
 jni/CMakeFiles
 jni/Makefile
 jni/cmake_install.cmake
-jni/release
```

Comment:
do we need jni/build path in the gitignore?

Reply:
on my local environment it seems to already ignoring it with the existing rules, but I can add it there just to be sure.

Reply:
I added it, and took the liberty of removing exclusions that are no longer relevant :)